ppa/archive/templates/archive/snippets/pagination.html

```diff
@@ -21,23 +21,23 @@
 
                 {#  always display current page, marked as active #}
                 {% if number == page_obj.number %}
-                    <a class="ui active basic button" href="?{% querystring_replace page=number %}">{{ number }}</a>
+                    <a title="page {{number}}" class="ui active basic button" href="?{% querystring_replace page=number %}">{{ number }}</a>
```

Comment:
not specific to your updates, but any thoughts on how to simplify the redundant code here without confusing the logic?

Reply:
I guess on a basic level we could just abstract a single button to a template that receives some information like if it's an arrow or a number, what number, what link it is, etc.

Then maybe do a little more work in the view to build a well-formed collection of button objects that we pass to that template and iterate it?